templates/shopapi/base.html

```diff
@@ -1,18 +1,28 @@
 {% load static %}
```

Comment:
@vasyltsar I would suggest to move this file to templates directory. templates/base.html. then you need to edit all templates and expand the base file like "../base.html"